src/docs/admin/references/properties/scheduler.properties

```diff
@@ -108,6 +108,10 @@ pa.scheduler.policy=org.ow2.proactive.scheduler.policy.ExtendedSchedulerPolicy
 # Defines the maximum number of tasks to be scheduled in each scheduling loop.
```

Comment:
All the changes below are related to the win agent patch ?

Reply:
the scheduler.properties file is automatically updated with the latest version downloaded from the scheduling project when the documentation gradle build is executed.